src/braket/tasks/gate_model_quantum_task_result.py

```diff
@@ -126,17 +123,16 @@ def get_value_by_result_type(self, result_type: ResultType) -> Any:
         except KeyError:
             raise ValueError(
                 "Result type not found in result. "
-                + "Result types must be added to circuit before circuit is run on device."
+                "Result types must be added to circuit before circuit is run on device."
```

Comment:
Is there a reason these two strings are separate instead of one string?

Reply:
Max line length